docs/whats_new/version_ongoing.rst

```diff
@@ -1,2 +1,6 @@
 Version ongoing
 ---------------
+
+New serial connector
+^^^^^^^^^^^^^^^^^^^^
+Added new connector for serial communication.
```

Comment:
Maybe add the name of ? like that we do not need to search everywhere the APIs of it?